src/core/service/notifications/clair/handler.go

```diff
@@ -64,8 +64,8 @@ func (h *Handler) Handle() {
 			ns[vuln.NamespaceName] = true
 		}
 	}
-	if new := notification.New; new != nil {
-		if vuln := new.Vulnerability; vuln != nil {
+	if newNotification := notification.New; newNotification != nil {
```

Comment:
I'm not a big fan of long variable name.  Could you let me know why this is necessary?